core/drivers/sub.mk

```diff
@@ -1,5 +1,7 @@
 srcs-$(CFG_PL011) += pl011.c
 srcs-$(CFG_GIC) += gic.c
+srcs-$(CFG_GPIO) += gpio.c
+srcs-$(CFG_PL061) += pl061_gpio.c
```

Comment:
Should we add

    #ifdef CFG_PL061
    $(call force,CFG_GPIO,y,required by CFG_PL061) 
    #endif